### ISSUE TITLE
Accessibility issue with category dropdown

### ISSUE THREAD

**Issue by MikeMorkes** [Automattic/WP-Job-Manager#1895]
**Describe the bug**
When running an accessibility audit on a wordpress page using the WP Job Manager shortcode "[jobs per_page="10" job_types="Full Time, Part Time, Temporary"]", the dropdown menu for Categories triggers a accessibility issue in the audit. Audit error message is:

_Elements with [role] that require specific children [role]s, are missing.
Some ARIA parent roles must contain specific child roles to perform their intended accessibility functions._

**Failing Elements**
span.select2-selection.select2-selection--single

This element has aria-labelledby assigned to it, but it points to another element that doesn't exist on the page.

**To Reproduce**
1. Go to page using the shortcode.
2. Open Dev Tools and run an audit.
3. See error in audit summary.

**Expected behavior**
The element should either a) have its own aria-label, or b) the missing element needs to be put back in the html code with an appropriate label.

**WordPress Environment**
- WordPress Version:  5.2.3
- WP Job Manager Version: 1.33.5
- PHP Version: tested with 5.6.4 and 7.3

**Comment by jom** [comment]
Thanks for reporting this issue. We use Select2 for this behavior, which has some known accessibility issues. They started working on them in the latest release and I just updated the version included in the plugin (in the latest 1.34.0), but it looks like I still see some warnings. 

PR with latest fixes https://github.com/select2/select2/pull/5582
Master issue https://github.com/select2/select2/issues/3735

WooCommerce supplements Select2's with its own augmented version. That might be necessary if more progress isn't made.
